docs/build_node.md

```diff
@@ -212,19 +191,12 @@ final channel_manager = ChannelManager.constructor_new(
 byte[] serialized_channel_manager = // <insert bytes you would have written in 
                                     // following the later step "Persist 
                                     // channel manager">
-Result_C2Tuple_BlockHashChannelManagerZDecodeErrorZ channel_manager_read_result =
-  UtilMethods.constructor_BlockHashChannelManagerZ_read(serialized_channel_manager, 
-  keys_manager.as_KeysInterface(), fee_estimator, chain_monitor.as_Watch(), 
-  tx_broadcaster, logger, UserConfig.constructor_default(), channel_monitors);
-
-// Assert we were able to read successfully.
-assert channel_manager_read_result instanceof 
-    Result_C2Tuple_BlockHashChannelManagerZDecodeErrorZ
-    .Result_C2Tuple_BlockHashChannelManagerZDecodeErrorZ_OK;
-
-final channel_manager = ((Result_C2Tuple_BlockHashChannelManagerZDecodeErrorZ
-    .Result_C2Tuple_BlockHashChannelManagerZDecodeErrorZ_OK) channel_manager_read_result)
-    .res.b;
+ChannelManagerConstructor channel_manager_constructor = new ChannelManagerConstructor(
+  serialized_channel_manager, channel_monitors.toArray(new byte[1][]),
```

Comment:
Could we add a line:
`final channel_monitors = channel_monitors.toArray(new byte[1][])` with a comment above the line describing what's happening in it?